chpl/chpl-service/src/main/java/gov/healthit/chpl/FeatureList.java

```diff
@@ -9,4 +9,5 @@ private FeatureList() {
     public static final String ERD_PHASE_2_GRACE_PERIOD_END = "erd-phase-2-grace-period-end";
     public static final String ERD_PHASE_3 = "erd-phase-3";
     public static final String SUBSCRIPTIONS = "subscriptions";
+    public static final String SSO = "sso";
```

Comment:
Make sure to add this to all environments (I happened to look at prod and did not see it there)